React/Getting started with React.md

```diff
@@ -1,12 +1,20 @@
 # Getting started with React
 
-### React
+## React
 
-- intro
+[React](https://reactjs.org/) is not a Framework, it is a JavaScript library for building user interfaces.
+with a declarative programming paradigm for building component based systems which could be reused anywhere.
 
-### Next.js Framework
+In the next sections (no pun intended) we will cover some topics related to React ecosystem used inside our company.
+
+## Next.js Framework
```

Comment:
I would suggest to have Next.js as a separate chapter in the Handbook.

Reply:
Done!